Source/Views/RepositoryCell/RepositoryCell.swift

```diff
@@ -11,7 +11,9 @@ import SwiftUI
 struct RepositoryCell: View {
 
     @State var viewModel: RepositoryCellViewModel
-    @State var isNotifying: Bool = true
+    @State var isNotifying: Bool
+
+    @ObservedObject var notifiedReposObservable = NotifiedRepositoryObservable()
```

Comment:
This works fine but to improve this, current we have two separated instances of `NotifiedRepositoryObservable`, I think we can use `@Binding` for `isNotifying` which is being passthrough from `SearchRepoScreen`'s `notifiedReposObservable`. You can take a look at this for fun (and make it more Combine-ish)